Lab 5

d3nik opened the pull request.

d3nik committed:
Patch 5.1

– added close icon svg;
– added backdrop modal window;
– added modal.js file;
– added some styles for modal window.

d3nik committed:
Patch 5.2

– stylized modal window;
– added transition for open/close modal window;
– added active nav-menu-link styles;

d3nik committed:
Patch 5.2

– added overlay text for portfolio cards;
– added bottom overlay for occupation images;
– fixed transition animations for hover/focus;

vozniuk-lntu reviewed (approved):
1. прижми донизу, щоб не було два білих пікселя https://prnt.sc/lJWvrO-XFMyO
2. ховер кнопочки закрити модалку
3. > .overlay-text там без маржинів, просто падінги
4. у футері ссилки теж мають мати ховер
5.

d3nik committed:
Patch 5.3 (hotfix)

– fixed underline of active page nav-menu;
– added hover/focus on .modal close-button;
– changed margins to padding on .overlay-text;
– .footer-contacts a have hover and focus.
